### PR TITLE
Document retriggering push events in git with PAC triggers

### PR DESCRIPTION
**Pull request by arewm**
This is currently the most reliable method of retriggering as some of the other methods have been hitting issues with all of the variables not being defined.

### CODE REVIEW

**Comment by arewm** [comment]
Replaces https://github.com/redhat-appstudio/docs.appstudio.io/pull/254

File: docs/modules/ROOT/pages/how-tos/configuring/rerunning.adoc [hunk 1]
```diff
@@ -14,11 +14,33 @@ See also xref:/how-tos/testing/integration/rerunning.adoc[Retriggering integrati
...
+== Retriggering a post-merge build from from GitHub
+
+.**Prerequisite**
+
+- You have already merged a pull request, but the subsequent build failed, prompting a need for retriggering.
+
+.**Procedure**
+
+. On the GitHub UI, navigate to the latest commit on your branch that that you want to rebuild.
+. Comment `retest` on the commit.
```
Comment (ralphbean) [inline docs/modules/ROOT/pages/how-tos/configuring/rerunning.adoc:32]:
```suggestion
. Comment `/retest` on the commit.
```

**Review by ralphbean — CHANGES_REQUESTED** [review]
One typofix needed.

**Review by ralphbean — APPROVED** [review]

**Comment by ralphbean** [comment]
Thanks @arewm !

**Pull request merged**